ibm/mas_devops/roles/appconnect_install/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# MAS 8.4 uses AppConnect channel subscription v1.4 and license id = L-APEH-BSVCHU
+# MAS 8.5 uses AppConnect channel subscription v2.0 and license id = L-KSBM-BZWEAT
+# MAS 8.6 uses AppConnect channel subscription v2.0 and license id = L-KSBM-C37J2R
+# MAS 8.7 uses AppConnect channel subscription v3.0 and license id = L-KSBM-C87FU2
+
+# Define where AppConnect will be installed
+appconnect_namespace: "{{ lookup('env', 'APPCONNECT_NAMESPACE') | default('ibm-app-connect', true) }}"
+# v3.0 channel is used in MAS 8.7, v2.0 channel is used in older MAS versions
+appconnect_channel: "{{ lookup('env', 'APPCONNECT_CHANNEL') | default('v3.0', true) }}"
+appconnect_dashboard_name: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('dashboard-12020r2', true) }}"
+appconnect_license_id: "{{ lookup('env', 'APPCONNECT_DASHBOARD_NAME') | default('L-KSBM-C87FU2', true) }}"
```

Comment:
Is there a special meaning to this default dashboard name: `dashboard-12020r2` ... is that a dashboard that ships with AppConnect as standard or something?  Could you add some details about the significance of this default in the README.